### ISSUE TITLE
No games launching, mismatch with md5?

### ISSUE THREAD

**Issue by stuffu** [libretro/libretro-uae#532]
Running WHDLoad games with PUAE 4.5 in Retroarch, on Xbox Series S.

I have tried launching a variety of games, but nothing loads. It just stays at kickstart with a repeating glitching sound. Just launching the core without any game show the very same screen and the same glitching sound.

All settings are default and all kickstarts match md5 requirements. Kickstarts are present, but there is an (!) in front of every kickstart. The md5 is displayed under the kickstarts, even if there is a 100% match. I think this is the issue but I have failed to found a solution to it.

**Comment by sonninnos** [comment]
No, that is not the issue. Seeing the Kickstart screen means the Kickstart is working. RA info screen does not check any hashes, it only shows them, and (!) is normal, like with every other core.

Something is off with the content then, but hard to guess without a log.

The repeating sound is a feature, not a glitch. It means there is no floppy disk inserted, and it can be silenced.

The issue is a bit too forum help desk post material and not something belonging here though.

**Comment by stuffu** [comment]
I know it’s not intended to be support here and I have tried official channels without luck.

The WHD content is “official” and works fine with other UAE forks. Tried both a1200, a500 and CD32 content.

Being a long term user of both original amiga and emulators I can tell the glitching sound is not near how the disk drive in amiga sounds like. There is obviously something off here.

can’t provide any logs unfortunately, but to rule out something (a bug), is there anything else I can test?

**Comment by sonninnos** [comment]
I can't think of anything else useful except the log. Something up with the paths perhaps..?
